src/deepsparse/v2/operators/operator.py

```diff
@@ -71,6 +71,18 @@ def __call__(
         :param kwargs: kwargs when not initializing from an instantiated schema
         :return: operator output
         """
+        name = None
```

Comment:
When we have multiple prompts, we copy the inference state for each prompt/subgraph execution. This allows each subgraph to update its specific subgraph, independent of the other subgraphs. We use the original inference state when we join the prompts and return to the normal single graph execution (i.e. not the sub-graph specific inference states) Since the middleware is stored in inference state, it seems like the individual operator timing will be off for this case?